IMC_WebDev/#Driving_the_car_and_flowing_the_water_with_CSS/blog.md

```diff
@@ -1,16 +1,17 @@
 # Introduction to Web Animations using CSS
 
-In this blog, we are going to keep building our mini city. We will have a car going along the road and the bridge, and a boat going under the bridge.
+In this blog, we are going to keep building our mini city. We will have a car going along the road and the bridge as well as a boat going under the bridge.
 
 ![](https://i.imgur.com/9FFPgkl.gif)
 
 
 
 ## Build the car and the boat
 
-First, let's build our car and boat with HTML and CSS like we did for the ground and buildings previously.
-The car is a rectangle box and the boat is a rectangle box with a triangle in the front. Both of them has a shadow to help show the 3D effect.
-So we need to have 2 divs for the car and the boat, and in each of them, we will have some children divs inside.
+First, let's build our car and boat with HTML and CSS, just like we did for the ground and buildings before.
+The car is a rectangle box, and the boat is a rectangle box with a triangle in the front. Both of them have a shadow to help show the 3D-effect.
+So we need to have two divs for the car and the boat, and inside each of them, we will have some children divs. Children divs are divs inside a parent div. They are associated to their parent and can be easily aligned with their parent div. The two children divs under the car are to hold the left and front side rectangles while the three children divs under the boat are to hold the triangle in the front and the left and front side rectangles.
```

Comment:
If you're assuming your audience is more new to CSS, you could go in-depth as to what "easily aligned" means to make sure they understand the connection between children and parent divs.